venom/python/AudioProcessor/PyAudioProcessor.cpp

```diff
@@ -0,0 +1,118 @@
+#include "include/PyAudioProcessor.h"
+
+#include <thread>
+#include <utility>
+
+#include "spdlog/spdlog.h"
+
+PyAudioProcessor::PyAudioProcessor(std::unique_ptr<py::object> clsInstance)
+        : juce::AudioProcessor(),
+          instance(std::move(clsInstance)) {
+}
+
+void PyAudioProcessor::prepareToPlay(double sampleRate, int samplesPerBlock) {
+    spdlog::debug("[Prepare To Play]");
+
+    py::gil_scoped_acquire acquire;
+    instance->attr("prepareToPlay")(sampleRate, samplesPerBlock);
+}
+
+void PyAudioProcessor::releaseResources() {
+    spdlog::debug("[Release Resources]");
+
+    py::gil_scoped_acquire acquire;
+    instance->attr("releaseResources")();
+}
+
+// TODO port BusesLayout
```

Comment:
Why are PyAudioProcessor and PyAudioProcessorEditor defined both here and in boilerplate project?